web/src/service-worker/index.ts

```diff
@@ -26,8 +33,16 @@ sw.addEventListener('fetch', (event) => {
     return;
   }
   const url = new URL(event.request.url);
-  if (/^\/api\/assets\/[a-f0-9-]+\/(original|thumbnail)/.test(url.pathname)) {
+  if (APP_RESOURCES.includes(url.pathname)) {
+    event.respondWith(appResources(url, event));
+  } else if (/^\/api\/assets\/[a-f0-9-]+\/(original|thumbnail)/.test(url.pathname)) {
     event.respondWith(immichAsset(url));
+  } else if (
+    /^(\/(link|auth|admin|albums|archive|buy|explore|favorites|folders|maps|memory|partners|people|photos|places|search|share|shared-links|sharing|tags|trash|user-settings|utilities))(\/.*)?$/.test(
+      url.pathname,
```

Comment:
Why is this an `if`? Which paths do you not want it to render if it isn't cached?